packages/common/mui/src/lib/bar-chart/bar-chart.tsx

```diff
@@ -20,220 +13,111 @@ import {
-        margin={{ top: 40, right: 10, left: 10, bottom: 20 }}
-      >
-        <CartesianGrid horizontal={true} vertical={false} />
-        <XAxis
-          dataKey="timestamp"
-          axisLine={false}
-          tickLine={false}
-          type={xAxisTickFormat === 'date' ? 'category' : 'number'}
-          // domain={[
-          //   (dataMin: number) => dataMin - ONE_WEEK,
-          //   (dataMax: number) => dataMax + ONE_WEEK,
-          // ]}
-          tickFormatter={xAxisTickHandler}
-          // tick={<XAxisTick xAxisTickFormat={'date'} />}
-          tickCount={3}
-        />
-        <YAxis
-          tickLine={false}
-          axisLine={false}
-          tickFormatter={(v: number) => yAxisTickHandler(v)}
-        />
-        <Tooltip
-          wrapperStyle={{ outline: 'none' }}
-          content={
-            <BarChartToolTip barChartToolTipData={barChartToolTipData} />
-          }
-          cursor={{ fill: 'transparent' }}
-          position={{ y: 0 }}
-        />
-        {/* <Bar
-          dataKey="dataPointTwo"
-          stackId="a"
-          fill={barChartStyles?.dataSetTwo.lineColor}
-        />  */}
-        <Bar
-          dataKey="dataPointOne"
-          // stackId="a"
-          fill={theme.palette.primary.light}
-          radius={[8, 8, 0, 0]}
-        />
-      </RechartsBarChart>
-    </ResponsiveContainer>
+    <Box>
+      {noChartData ? (
+        <Box sx={{ marginLeft: theme.spacing(3), marginTop: theme.spacing(6) }}>
+          <FormattedMessage
+            defaultMessage={'You have no active orders or historical data.'}
+          />
+        </Box>
+      ) : (
+        <ResponsiveContainer width="100%" height={300}>
+          <RechartsBarChart
+            barSize={7}
+            data={barChartData}
+            margin={{ top: 30, right: 10, left: 10, bottom: 0 }}
+          >
+            <CartesianGrid
+              vertical={false}
+              stroke={theme.palette.borders.paper}
+            />
+            <XAxis
+              dataKey="timestamp"
+              tickLine={false}
+              tickFormatter={formatDate}
+              axisLine={{ stroke: theme.palette.borders.paper }}
+              interval={0}
+            />
+            <YAxis
+              tickLine={false}
+              axisLine={false}
+              tickFormatter={(v: number) => yAxisTickHandler(v)}
+            />
+            <Tooltip
+              wrapperStyle={{ outline: 'none' }}
+              content={<BarChartToolTip barConfig={barConfig} />}
+              cursor={{ fill: 'transparent' }}
+              position={{ y: 0 }}
+            />
+
+            {barConfig.map(({ dataKey, fill }, index) => (
+              <Bar
+                key={index}
+                dataKey={dataKey}
+                fill={fill}
+                radius={[8, 8, 0, 0]}
+              />
+            ))}
+          </RechartsBarChart>
+        </ResponsiveContainer>
+      )}
+    </Box>
   );
 };
 
+// NOTE*
```

Comment:
Not sure what this comment refers to?

Reply:
If you want to have the bars in the bar chart be on top of one another you can pass the same stackId

Reply:
Just a note to remember that since we had other mocks with stacked bars